assets/src/modules/Search.js

```diff
@@ -226,21 +224,31 @@ export default class Search {
                         lizMap.addMessage(lizDict['externalsearch.ignlimit'], 'warning', true);
                         break;
                     }
-                    let mapExtent4326 = transformExtent(this._map.getView().calculateExtent(), this._map.getView().getProjection().getCode(), 'EPSG:4326');
-                    let queryParam = '?text=' + $('#search-query').val() + '&type=StreetAddress&maximumResponses=10&bbox=' + mapExtent4326;
-                    $.getJSON(encodeURI(service + queryParam), data => {
-                        let text = '';
-                        let count = 0;
-                        for (const result of data.results) {
-                            var lab = result.fulltext.replace(labrex, '<strong class="highlight">$1</strong>');
-                            text += `<li><a href="#${result.x},${result.y},${result.x},${result.y}" data-wkt="POINT(${result.x} ${result.y})">${lab}</a></li>`;
-                            count++;
-                        }
-                        if (count == 0 || text == '') {
-                            text = '<li>' + lizDict['externalsearch.notfound'] + '</li>';
+                    $.get(service
```

Comment:
Could you use `fetch` instead please?

Reply:
I homogenize the code and fix the issue.

@nboisteault once it is merged and backported, could you provide a refactoring ?